sensor.cpp

```diff
@@ -151,7 +151,7 @@ bool SensorFingerprint::hasOutCluster(quint16 clusterId) const
 Sensor::Sensor() :
     Resource(RSensors),
     m_deletedstate(Sensor::StateNormal),
-    m_mode(ModeTwoGroups),
+    m_mode(ModeScenes),
```

Comment:
Tricky ;)

I was considering this myself recently but hold back since I wasn't sure if that doesn't break some switches.